### DISCUSSION TITLE
Adding ore variants to modded dimensions (with help of a datapack)?

### DISCUSSION THREAD

**Discussion by ratemisia** [lukebemish/excavated_variants#197] [Q&A] [answered]
The previous discussion at #153 mentions that if a datapack or mod is used to add ores to the Nether, Excavated Variants can handle changing the ore veins to match the stone type they spawn in (so netherrack, etc. in that case). Does Excavated Variants check and replace ores in modded dimensions in the same way, or would one need to make a custom mod to generate ores matching the stone types in modded dimensions?

**Comment by lukebemish** [comment, marked as answer]
Excavated Variants accomplishes this through using a biome modifier to add a feature which accomplishes this replacement to every biome. If the modded dimensions have somewhat normal worldgen - which most should - then this will work just fine there, yes.

**Reply by ratemisia** [reply]
Thank you for the quick reply! Glad to hear it should work, as I'm fairly sure the dimensions I'm working with have normal worldgen - this makes my job a lot easier.
